### PR TITLE
Added student card recognition using OCR.

### PR DESCRIPTION
**Pull request by avizen-j**
Added file upload to settings tab.
Added a feature where upload student card is being validated using Vision API.

### CODE REVIEW

File: University-advisor-web/Controllers/SettingsController.cs [hunk 1]
```diff
@@ -122,5 +125,14 @@ public IActionResult ChangeStatus(UserModel model)
...
         }
+
+        [HttpPost]
+        public IActionResult UploadFile(UserModel model)
+        {
+            var rec = new CardRecognition(_logger);
+            var result = rec.StartStudentCardValidation(model.File);
+            _errorHandler.ShowError(this, result.Information, "Information");
+            return RedirectToAction("Index", model);
+        }
```
Comment (MartinCepulionis) [inline University-advisor-web/Controllers/SettingsController.cs:136]: What will be shown to user if the validation failed? Shouldn't there be any error handling?

Comment (avizen-j) [reply University-advisor-web/Controllers/SettingsController.cs:136]: For some reason file was not saved. I will fix that immediately.

File: University-advisor-web/Models/ValidationResponse.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            }
+            else if (rate == -1)
+            {
+                Information = "Something is not right with Vision API. Try again later";
+                Successful = false;
+            }
+            else
+            {
+                Information = "Uploaded document is invalid. Match is only: " + rate + "%";
+                Successful = false;
```
Comment (MartinCepulionis) [inline University-advisor-web/Models/ValidationResponse.cs]: Maybe we should move these hardcoded messages to our separate class Messages?

Comment (avizen-j) [reply University-advisor-web/Models/ValidationResponse.cs]: Great point.

File: University-advisor-web/Tools/CardRecognition.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                }
+                else
+                {
+                    validationResponse.SetInformation(rate);
+                    return validationResponse;
+                }
+            }
+            catch (Exception e)
+            {
+                _logger.Log("Something is not right with Vision API" + e.StackTrace, "ERROR");
```
Comment (MartinCepulionis) [inline University-advisor-web/Tools/CardRecognition.cs]: I think this message could be also moved.

**Pull request merged**